test/apiv2/10-images.at

```diff
@@ -440,4 +440,19 @@ t GET "libpod/events?stream=false&since=$START"  200  \
   .Actor.Attributes.name="localhost:5000/idonotexist" \
   .Actor.Attributes.error~".*connection refused"
 
+# test empty RepoTags and RepoDigests is an empty array
+IIDFILE=$(mktemp)
+podman image build --iidfile $IIDFILE -<<EOF
+from alpine
```

Comment:
can you use $IMAGE here instead of alpine.
I wonder if the issue is that alpine is leaked into the follow up tests that don't expect that extra image